cross/libvpx/Makefile

```diff
@@ -0,0 +1,45 @@
+PKG_NAME = libvpx
+PKG_VERS = 1.5.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/webmproject/libvpx/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = http://www.webmproject.org/code/
+COMMENT  =
+
+CONFIGURE_ARGS = --disable-sse --disable-sse2 --disable-sse3 --disable-ssse3 --disable-sse4_1 --enable-shared
+CONFIGURE_ARGS += --enable-vp8 --enable-vp9 --disable-examples --disable-docs --disable-unit_tests
+CONFIGURE_ARGS += --enable-postproc --enable-vp9-postproc --enable-pic  --prefix=$(INSTALL_PREFIX)
+
+include ../../mk/spksrc.cross-cc.mk
+
+ENV = LD= CPP= NM= CC= RANLIB= CXX= AR= STRIP= OBJDUM= CFLAGS="$(CFLAGS)" CPPFLAGS= LDFLAGS="$(LDFLAGS)" CXXFLAGS= READELF= AS= OBJDUMP= DESTDIR=$(INSTALL_DIR) CROSS=$(TC_PATH)$(TC_PREFIX)
```

Comment:
Without this line, I can compile armv5, armv7 and ppc but it failed for x86 and x64 when it tries to use `yasm` :

        [AS] vpx_ports/emms.asm.o
    Assembler messages:
    Error: can't open elf64 for reading: No such file or directory
    vpx_ports/emms.asm:2: Error: no such instruction: `copyright (c) 2010 The WebM project authors. All Rights Reserved.'
    vpx_ports/emms.asm:4: Error: no such instruction: `use of this source code is governed by a BSD-style license'
    vpx_ports/emms.asm:5: Error: no such instruction: `that can be found in the LICENSE file in the root of the source'
    vpx_ports/emms.asm:6: Error: no such instruction: `tree. An additional intellectual property rights grant can be found'
    vpx_ports/emms.asm:7: Error: junk `file PATENTS.  All contributing project authors may' after expression
    vpx_ports/emms.asm:7: Error: operand size mismatch for `in'
    vpx_ports/emms.asm:8: Error: no such instruction: `be found in the AUTHORS file in the root of the source tree.'